.ruby-version

```diff
@@ -1 +1 @@
-2.4.2
+2.6.5
```

Comment:
Fun note: The tests on master have been failing since last year because Ruby 2.4 does not support &.

Reply:
:awkward: